sdk/cosmos/azure-spring-data-cosmos-core/README.md

```diff
@@ -595,25 +573,78 @@ public class MultiDatasourceApplication implements CommandLineRunner {
 
 Beta version built from `master` branch are available, you can refer to the [instruction](https://github.com/Azure/azure-sdk-for-java/blob/master/CONTRIBUTING.md#nightly-package-builds) to use beta version packages.
 
-
 ## Troubleshooting
 
+### General 
+
 If you encounter any bug, please file an issue [here](https://github.com/Azure/azure-sdk-for-java/issues/new).
 
 To suggest a new feature or changes that could be made, file an issue the same way you would for a bug.
 
-## Contributing
-
-Contribution is welcome. Please follow [this instruction](./CONTRIBUTING.md) to contribute code.
-
-## Code of Conduct
+### Enable Client Logging
+- Azure-spring-data-cosmos uses SLF4j as the logging facade that supports logging into popular logging frameworks such as log4j and logback. 
+For example, if you want to use spring logback as logging framework, add the following xml to resources folder. 
 
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+```xml
+<configuration>
+  <include resource="/org/springframework/boot/logging/logback/base.xml"/>
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+    <encoder>
+      <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n
+      </pattern>
+    </encoder>
+  </appender>
+  <root level="info">
+    <appender-ref ref="STDOUT"/>
+  </root>
+  <logger name="com.azure.cosmos" level="error"/>
+  <logger name="org.springframework" level="error"/>
+  <logger name="io.netty" level="error"/>
+</configuration>
+```
 
-### Data
+## Examples
+- Please refer to [sample project here][samples].
 
- This project collects usage data and sends it to Microsoft to help improve our products and services. Read our [privacy](https://privacy.microsoft.com/privacystatement) statement to learn more.
+## Next steps
+- Read more about azure spring data cosmos [here][azure_spring_data_cosmos_docs].
```

Comment:
The `azure_spring_data_cosmos_docs` link is missing.

Reply:
Good find, updated.